lightning/src/ln/channelmanager.rs

```diff
@@ -4134,6 +4213,15 @@ impl<'a, Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref>
 			}
 		}
 
+		let background_event_count: u64 = Readable::read(reader)?;
+		let mut pending_background_events_read: Vec<BackgroundManagerEvent> = Vec::with_capacity(cmp::min(background_event_count as usize, MAX_ALLOC_SIZE/mem::size_of::<BackgroundManagerEvent>()));
+		for _ in 0..background_event_count {
+			match <u8 as Readable>::read(reader)? {
+				0 => pending_background_events_read.push(BackgroundManagerEvent::ClosingMonitorUpdate((Readable::read(reader)?, Readable::read(reader)?))),
+				_ => return Err(DecodeError::InvalidValue),
+			}
+		}
```

Comment:
would it be easy to test deserialization for this?

Reply:
It is, at least indirectly, via the test_unconf_chan test update (which I also wanted cause I wasnt sure if the free-update-after-reload-causing-monitor-close-twice condition was gonna work). Would you prefer a unit test here as well?

Reply:
> would it be easy to test deserialization for this?

Oops thought I deleted this comment